content/docs/reactivesearch/v3/advanced/selectedfilters.md

```diff
@@ -60,7 +60,10 @@ Example uses:
 	}}
 />
 ```
-
+-   **resetToDefault** `boolean` [optional]
+    When set to true and clearAll functionality is utilised, then it would set the filter's value to its default set value instead of null/ undefined.
```

Comment:
```suggestion
    When set to true and clearAll functionality is utilised, then it would set the filter's value to its default set value(the `defaultValue` prop) instead of null/ undefined.
```